### ISSUE TITLE
adding freq 15/25/31 HZ option in games +custom resolution for all the games

### ISSUE THREAD

**Issue by Moksi81** [teknogods/TeknoParrotUI#797]
Hello is it possible to add the ability to choose which Freq 15/25/31 hz you can run your TP game  specifically  CRT TV 
Most games run windowed only 

& games that dont have a custom res option yet 

thanks alot

**Comment by Hydreigon223** [comment]
Some games also love to mess with various timing settings (CRTC and DDC) on the fly. Such examples are the Galeco PC based games at least according to [this source](https://github.com/mamedev/mame/blob/master/src/mame/gaelco/gaelcopc.cpp#L9).

**Comment by Redemp** [comment]
I'm going to try to explain the issue more in detail here.

On Windows it's nearly impossible to avoid some programs from choosing 31 kHz, even if 480i is available and 480p supposedly blocked.
This is because 99% of software ignore the refresh rate exists, and simply ask for width x height.
In that situation, Windows always finds a way to pull a vesa 640x480 mode from the drivers.
In other words, The if the program doesn't bother requesting a specific refresh for 640 x 480, Windows will apply 31 kHz

So with that said we need a function to select a resolution available in Windows from Teknoparrot.
That should solve the issue completely without relying on running those troublesome games in a window then use something like a borderless Window program.

**Comment by Redemp** [comment]
> Some games also love to mess with various timing settings (CRTC and DDC) on the fly. Such examples are the Galeco PC based games at least according to [this source](https://github.com/mamedev/mame/blob/master/src/mame/gaelco/gaelcopc.cpp#L9).

Yes that is correct but that is a non issue if you use GroovyMame or Retroarch with the Swirchres api activated. Another good example is games on playstation 1 where it switches between 480i to 240p boot during boot and also in-game. 

But I'm unsure if that is the issue here since many of the games available in Teknoparrot where released using LCDs.

**Comment by nixxou** [comment]
You can use third party app to set resolution and frequency before the game launch and restore it after, it's a non-issue.
